tests/test_dwarf.py

```diff
@@ -2059,3 +2059,13 @@ def test_language(self):
         self.assertEqual(
             dwarf_program(dies, lang=DW_LANG.BLISS).language, DEFAULT_LANGUAGE
         )
+        self.assertEqual(
+            dwarf_program(dies, lang=DW_LANG.C_plus_plus_14).language,
+            Language.CPP
+        )
+
+        tmp = dwarf_program(dies, lang=DW_LANG.C_plus_plus)
```

Comment:
Removing this temp variable causes a segfault, and I'm not sure why. Could that be another bug? I'm not sure where to begin looking for that. It seems like the parent object must be deleted while it's children is alive.

Reply:
https://github.com/osandov/drgn/commit/79f973007b2df7ecaacca40312a1b46befa7d53a